fd/scripts/download.py

```diff
@@ -0,0 +1,136 @@
+#
```

Comment:
general comment to the scripts below, should we introduce a running number to the filename (as we had for notebooks), to make clearer in which order they should be executed?

Reply:
I think if we want the scripts to be used within the notebook we can't :speak_no_evil:  
I.e, trying to import: 
`from fd.scripts.01_download import batch_download`
gives: 
`SyntaxError: invalid decimal literal`

The current solution is a bit hacky/ugly currently. All the scripts have the same form with two parts: 
* argument parsing 
* the core function that actually does stuff

The core functions could be moved into separate file(s) (which could be imported) and have the scripts just be the argument parsing and the calling of the core function (these could be named whatever as they wouldn't be importable modules)

Not sure if this is something we want to spend time on tho. The order is pretty clear both in the documentation and in the end2end notebook.